templates/index.html

```diff
@@ -229,7 +229,7 @@ <h3>אודות ANYWAY</h3>
             ANYWAY היא הדרך שלנו למנוע את התאונה הבאה. ביחד, נהגים, הולכי רגל, אזרחים ומקבלי החלטות נאבק בתאונות הדרכים.
             <br/>
             <br/>
- ANYWAY מציגה על גבי מפה תאונות דרכים עם נפגעים ע״ב נתוני הלשכה המרכזית לסטטיסטיקה, ונתונים המוזרמים בזמן אמת באדיבות ארגון ״איחוד הצלה״. בהמשך, יתווסף מידע נוסף למפה וכן תתאפשר יכולת לפתח דיונים מבוססי מיקום על פתרונות אפשריים והתארגנות אזרחית. באמצעות ידע זמין, נגיש ושקוף אנו שואפים להעלות את המודעות לנושא תאונות הדרכים ולהעצים את יכולת ההשפעה של כל אחד ואחת על הסביבה הקרובה- הבית, השכונה, בית הספר של הילדים, מקום העבודה ועוד- על ידי פעולה משותפת בקהילה. אנחנו מאמינים שבעזרת ידע ציבורי קיים המוצג באתר יווצרו ידע ותובנות חדשים. שיתוף הידע הזה יאפשר לחברה האזרחית להתארגן לפעולה אקטיבית ולהזרים הצעות, רעיונות והתראות בחזרה אל הרשויות. 
+ANYWAY מציגה על גבי מפה תאונות דרכים עם נפגעים ע״ב נתוני הלשכה המרכזית לסטטיסטיקה, ונתונים המוזרמים בזמן אמת באדיבות ארגון ״איחוד הצלה״. בהמשך, יתווסף מידע נוסףלמפה וכן תתאפשר יכולת לפתח דיונים מבוססי מיקום על פתרונות אפשריים והתארגנות אזרחית. באמצעות ידע זמין, נגישושקוף אנו שואפים להעלות את המודעות לנושא תאונות הדרכים ולהעצים את יכולת ההשפעה של כל אחד ואחת על הסביבההקרובה- הבית, השכונה, בית הספר של הילדים, מקום העבודה ועוד- על ידי פעולה משותפת בקהילה. אנחנו מאמינים שבעזרתידע ציבורי קיים המוצג באתר יווצרו ידע ותובנות חדשים. שיתוף הידע הזה יאפשר לחברה האזרחית להתארגן לפעולהאקטיבית ולהזרים הצעות, רעיונות והתראות בחזרה אל הרשויות.
```

Comment:
Whitespace?

Reply:
fixed some malformed characters

Reply:
Looks like a missing space.

Reply:
will be fixed in a future PR, much more work needed there